### DISCUSSION TITLE
Error shows in all pom.xml files each time open DBeaver on eclipse

### DISCUSSION THREAD

**Discussion by shawnsp** [dbeaver/dbeaver#12638] [Q&A]
Environment: 
macOS Big Sur
Eclipse 2021-03
DBeaver 21.1.0

DBeaver condition:
Build and run successfully.

Issue:
After closing and restarting Eclipse on DBeaver, all pom.xml files give me errors. 
Reinstall Eclipse and rebuild DBeaver would fix the problem, but if I restart eclipse again this issue reoccurs.

Need help with this. 
Currently, each time I work on DBeaver, I have to reinstall Eclipse as well as DBeaver.
Thank you.

Screenshots:
![100](https://user-images.githubusercontent.com/25361923/119909905-db044380-bf0a-11eb-9ac1-b9ceff6b898b.png)
![101](https://user-images.githubusercontent.com/25361923/119909919-dfc8f780-bf0a-11eb-97b9-c1197b8a346b.png)

**Comment by LonwoLonwo** [comment]
Hello @shawnsp 
Have you tried "Update project" button?

![2021-05-31 15_31_27-](https://user-images.githubusercontent.com/45152336/120193973-b8816b80-c225-11eb-8ae1-620fb1e266ed.png)

By the way, for example, I use Java 11 version for the project, and not 15, perhaps the problem may hide in this.

And thanks for JSONUtils tests)
